modules/project-structure/01-project-structure.qmd

```diff
@@ -0,0 +1,180 @@
+---
+title: Project structure
+subtitle: 'In this topic, you will learn about the project structure and how to manage your project in R properly.'
+title-block-banner: yes
+engine: knitr
+filters:
+  - webr-teachr
+execute: 
+  echo: false
+---
+
+```{r}
+knitr::opts_chunk$set(
+    fig.width = 8,
+    fig.height = 8,
+    fig.align = "center")
+```
+
+# Paths {#sec-path}
+
+The **paths** are used to identify the location of a file, computer, device, or web page. You could think of it as a direction pointing to the destination.
+
+## What does it look like?
+
+### File path
+
+The path may look different depending on the system you are using. If we want to locate a folder called **learnR**, this is what it may look like.
+
+**Windows:**
+
+`C:\Desktop\learnR\data\penguins.csv`
+
+**MacOS & Linux:**
+
+`/Users/Desktop/learnR/data/penguins.csv`
+
+### Webpage
```

Comment:
Maybe break this out into a tip - for example, paths can also reference places on other computers. Most commonly this is used when we get data from the internet with a URL.